### ISSUE TITLE
Always recompiling

### ISSUE THREAD

**Issue by rif** [unixorn/zsh-quickstart-kit#107]
When using .zgen-local-plugins files I always get this message when I start a new shell (without touching the file). I tried with the default example and got the same issue.

```shell
.zgen-local-plugins updated; creating a new init.zsh from plugin list in /home/rif/.zgen-local-plugins
-- zgenom: Creating `/home/rif/.zgenom/init.zsh` ...
-- zgenom: Compiling files ...
```

I am using zsh 5.8 (x86_64-pc-linux-gnu) on arch linux

**Comment by mingaldrichgan** [comment]
Duplicate of #106

**Comment by unixorn** [comment]
Thanks for the detailed report. @mingaldrichgan fixed this in #113, closing.
